notebooks/week3/03.model_serving_feature_lookup.py

```
@@ -0,0 +1,160 @@
+
+print("Response status:", response.status_code)
+print("Reponse text:", response.text)
+print("Execution time:", execution_time, "seconds")
+
+# COMMAND ----------
+
+wine_features = spark.table(f"{catalog_name}.{schema_name}.wine_features").toPandas()
+
+# COMMAND ----------
+
+wine_features.dtypes
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)